<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>460</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>460</width>
     <height>380</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#main{
	background-color: rgb(250, 250, 250);
    border: 1px solid rgb(196, 199, 206);
    border-radius: 6px;
    color: rgb(34, 34, 34);
}</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>5</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
color: rgb(164, 164, 164);
border: 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/icon/close.png</normaloff>:/resources/icon/close.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>5</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(164, 164, 164);
border: 0;
</string>
    </property>
    <property name="text">
     <string>时间设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeSave">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>120</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999999</width>
      <height>10000002</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 150, 219);
color: rgb(255, 255, 255);
border:  0;
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MiSans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 150, 219);
color: rgb(255, 255, 255);
border:  0;
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/icon/add.png</normaloff>:/resources/icon/add.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="timeList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>420</width>
      <height>220</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
